core/store/src/flat/store_helper.rs

```diff
@@ -126,10 +127,6 @@ pub fn remove_all_flat_state_values(store_update: &mut StoreUpdate, shard_uid: S
     remove_range_by_shard_uid(store_update, shard_uid, DBCol::FlatState);
 }
 
-pub fn remove_all_state_values(store_update: &mut StoreUpdate, shard_uid: ShardUId) {
```

Comment:
That was only used by `test_delete_trie_for_shard`, so I removed it together with the test.